src/vs/editor/contrib/snippet/snippetController2.ts

```diff
@@ -26,6 +27,7 @@ export interface ISnippetInsertOptions {
 	undoStopBefore: boolean;
 	undoStopAfter: boolean;
 	clipboardText: string | undefined;
+	overtypingCapturer: OvertypingCapturer | null;
```

Comment:
This is outside our guidelines and personal preference but I'd prefer `undefined` over `null` as it used for other members in snippet land.

Reply:
Sure. I'm not familiar enough with TypeScript yet, this is my first experience with it, so it's better to check me twice =). Used null because I saw it somewhere in other files. Decided to use undefined for value-types and null for class-types.